client/components/Quizlist.jsx

```diff
@@ -45,10 +42,12 @@ export const Quizlist = () => {
   return (
     <div className={classes.root}>
       <Box m={2}>
-        {quizzes.map((quiz, index) => (
+        {quizList.map((quiz) => (
           <Accordion expanded={expanded === quiz} onChange={handleChange(quiz)}>
```

Comment:
We should provide a `key` here for React optimization - `quiz.id` would be great

Reply:
Thank you. I will make sure to add in the keys for optimization!